Dockerfile

```diff
@@ -32,5 +34,10 @@ RUN mkdir /projects && \
 
 ADD etc/entrypoint.sh /entrypoint.sh
 
+RUN mkdir -p /var/tmp/containers/runtime && \
```

Comment:
could we merge this RUN with previous RUN instruction ?

Reply:
Reworked